relayer/relayer/src/connections/eth.rs

```diff
@@ -22,8 +22,9 @@ use tokio::sync::Mutex;
 use crate::config::Config;
 
 pub type EthConnection = Provider<Http>;
+pub type PersistentEthConnection = GasEscalatorMiddleware<EthConnection>;
```

Comment:
correct me if i'm wrong, bu there's nothing persistent in the connection object itself - just how you use it, so the name is a misnomer?

Reply:
Yes, it's about how it's used - do you propose to remove the alias?